matrix_sdk/src/lib.rs

```diff
@@ -41,11 +41,10 @@
 //! keys. If this is disabled and `encryption` support is enabled the keys will
 //! by default be stored only in memory and thus lost after the client is
 //! destroyed.
-//! * `unstable-synapse-quirks`: Enables support to deal with inconsistencies
-//! of Synapse in compliance with the Matrix API specification.
 //! * `markdown`: Support for sending markdown formatted messages.
 //! * `socks`: Enables SOCKS support in reqwest, the default HTTP client.
 //! * `sso_login`: Enables SSO login with a local http server.
+//! * `require_auth_for_profile_requests`: Matches the Synapse configuration with the same name
```

Comment:
Let's explain this a bit better so people don't need to chase the Synapse docs or config examples.